bindings/experimental/distrdf/python/DistRDF/Backends/__init__.py

```diff
@@ -0,0 +1,44 @@
+## @author Vincenzo Eduardo Padulano
+#  @author Enric Tejedor
+#  @date 2021-02
+
+################################################################################
+# Copyright (C) 1995-2021, Rene Brun and Fons Rademakers.                      #
+# All rights reserved.                                                         #
+#                                                                              #
+# For the licensing terms see $ROOTSYS/LICENSE.                                #
+# For the list of contributors see $ROOTSYS/README/CREDITS.                    #
+################################################################################
+
+import pkgutil
+import types
+import importlib
+
+
+def build_backends_submodules(parentmodule):
+    """
+    Helper function to create the submodules of the backends.
+    """
+    backends = ["Spark", ]
```

Comment:
Why having a concrete list and at the same time dynamically walking the available backends? Shouldn't it be one or the other?

Reply:
`walk_packages` also finds `Utils` and `Base` that we don't want to inject into the parent module. This was the fastest way that came to mind to filter only the actual backends we want

Reply:
I see, what I would do is only walk packages to discriminate: `Spark` is a package and so will be the other backends (they will have a subdirectory), `Utils` and `Base` are modules.

Reply:
Ok so I will turn around the logic using maybe `os.path.walk` and then `import_module` only inside the folders of the backends

Reply:
Update: `pkgutil.walk_packages` returns a named tuple with `ispkg` as the third element. We'll use that boolean to discriminate the backends packages